build/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM alpine:3.3
-RUN apk --update --no-cache add \
-	ca-certificates
+
+RUN apk add --no-cache ca-certificates && \
+	rm -rf /var/cache/apk/*
```

Comment:
I don't know. Two files are created in this directory after the `apk add` command and that is what they do in an example here: https://hub.docker.com/_/alpine/